packages/extension/public/fonts/fonts.css

```
@@ -17,6 +17,15 @@
        url('./OdibeeSans-Regular.ttf') format('truetype'), /* Safari, Android, iOS */
 }
 
+@font-face {
+  font-family: 'Oswald';
```

Stylelint (17.7.0)

[error] Expected no quotes around "Oswald" (font-family-name-quotes)

(font-family-name-quotes)